alica-essentials/LICENSE

```diff
@@ -0,0 +1,21 @@
+T License (MIT)
```

Comment:
can kill this. alica already has license which applies to full repo other than the license in alica_designer_runtime